tasks.py

```diff
@@ -229,48 +229,87 @@ def load_osm_streets(ctx, files=[]):
-        additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
-        additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
-        additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
-        # TODO take multiples oa files ?
-        run_rust_binary(
-            ctx,
-            "mimir",
-            "openaddresses2mimir",
-            files,
-            "--input {ctx.addresses.oa.file} \
-            --connection-string {ctx.es} \
-            {additional_params} \
-            --dataset {ctx.dataset}".format(
-                ctx=ctx, additional_params=additional_params
-            ),
-        )
+    if ctx.get("new-importer", False) is False:
+        addr_config = ctx.get("addresses")
+        if addr_config.get("bano", {}).get("file"):
+            logging.info("importing bano addresses")
+            conf = ctx.addresses.bano
+
+            additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
+            additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
+            additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
+            run_rust_binary(
+                ctx,
+                "mimir",
+                "bano2mimir",
+                files,
+                "--input {ctx.addresses.bano.file} \
+                --connection-string {ctx.es} \
+                {additional_params} \
+                --dataset {ctx.dataset}".format(
+                    ctx=ctx, additional_params=additional_params
+                ),
+            )
+        if addr_config.get("oa", {}).get("file"):
+            logging.info("importing oa addresses")
+            conf = ctx.addresses.oa
+            additional_params = _get_cli_param(conf.get("nb_threads"), "--nb-threads")
+            additional_params += _get_cli_param(conf.get("nb_shards"), "--nb-shards")
+            additional_params += _get_cli_param(conf.get("nb_replicas"), "--nb-replicas")
+            # TODO take multiples oa files ?
+             run_rust_binary(
+                ctx,
+                "mimir",
+                "openaddresses2mimir",
+                files,
+                "--input {ctx.addresses.oa.file} \
+                --connection-string {ctx.es} \
+                {additional_params} \
+                --dataset {ctx.dataset}".format(
+                    ctx=ctx, additional_params=additional_params
+                ),
+            )
+        return
+
+    output_csv = "output.csv.gz"
+
+    options = []
+    if ctx.addresses.get("bano", {}).get("file"):
+        options.append("--bano")
+        options.append(ctx.addresses.bano.file)
+    if ctx.addresses.get("oa", {}).get("file"):
+        options.append("--openaddresses")
+        options.append(ctx.addresses.oa.file)
+    if ctx.addresses.get("osm", {}).get("file"):
+        options.append("--osm")
+        options.append(ctx.addresses.osm.file)
```

Comment:
I think you forgot to download this file.

Changes introduced in https://github.com/QwantResearch/docker_mimir/pull/22 should ensure that we can call the download image without redownloading the same file over and over.

Reply:
Normally, "load_all" is called beforehand, either directly or through "load-in-docker-and-test". So we should have it at this point.